lib/features/scan_nfc/cubit/scan_nfc_cubit.dart

```diff
@@ -77,6 +81,9 @@ class ScanNfcCubit extends Cubit<ScanNfcState> {
           scanNFCStatus: ScanNFCStatus.error,
           coinAnimationStatus: CoinAnimationStatus.stopped));
       NfcManager.instance.stopSession();
+      AnalyticsHelper.logEvent(
```

Comment:
shall we also track mediumID when coin is scanned?

Reply:
Yes, I added it but in this case its an error so nothing was scanned from the NFC